src/pages/Registration/Steps/Documentation/index.tsx

```diff
@@ -14,7 +15,12 @@ function Documentation() {
   const methods = useForm<FormValues>({
     resolver: yupResolver(schema),
     defaultValues: doc
-      ? { ...(({ level, ...doc }) => doc)(doc) }
+      ? {
```

Comment:
Github auto-merged "active countries" & "hq country" commits in a way that breaks forms that use those fields (like this one), created a PR into this one to address this https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1796

Reply:
@misicnenad, this error is already fixed in #1790